docs/classad-attributes/machine-classad-attributes.rst

```diff
@@ -1090,316 +852,227 @@ Machine ClassAd Attributes
     Gives the amount of memory available for starting additional VM jobs
     on this machine, given in Mbytes. The maximum value is set by the
-    configuration variable ``VM_MEMORY``. :index:`VM_MEMORY`
-
-:index:`VM_Networking<single: VM_Networking; ClassAd machine attribute>`
-
-``VM_Networking``
+    configuration variable ``VM_MEMORY``. :classad-attribute:`VM_MEMORY`
     A boolean value indicating whether networking is allowed for virtual
     machines on this machine.
 
-:index:`VM_Type<single: VM_Type; ClassAd machine attribute>`
-
-``VM_Type``
+:classad-attribute:`VM_Type`
     The type of virtual machine software that can run on this machine.
     The value is set by the configuration variable ``VM_TYPE``
 
-:index:`VM_TYPE`
-:index:`VMOfflineReason<single: VMOfflineReason; ClassAd machine attribute>`
-
-``VMOfflineReason``
+:classad-attribute:`VM_TYPE`
     The reason the VM universe went offline (usually because a VM
     universe job failed to launch).
 
-:index:`VMOfflineTime<single: VMOfflineTime; ClassAd machine attribute>`
-
-``VMOfflineTime``
+:classad-attribute:`VMOfflineTime`
     The time that the VM universe went offline.
 
-:index:`WindowsBuildNumber<single: WindowsBuildNumber; ClassAd machine attribute>`
-
-``WindowsBuildNumber``
+:classad-attribute:`WindowsBuildNumber`
     An integer, extracted from the platform type, representing a build
     number for a Windows operating system. This attribute only exists on
     Windows machines.
 
-:index:`WindowsMajorVersion<single: WindowsMajorVersion; ClassAd machine attribute>`
-
-``WindowsMajorVersion``
+:classad-attribute:`WindowsMajorVersion`
     An integer, extracted from the platform type, representing a major
     version number (currently 5 or 6) for a Windows operating system.
     This attribute only exists on Windows machines.
 
-:index:`WindowsMinorVersion<single: WindowsMinorVersion; ClassAd machine attribute>`
-
-``WindowsMinorVersion``
+:classad-attribute:`WindowsMinorVersion`
     An integer, extracted from the platform type, representing a minor
     version number (currently 0, 1, or 2) for a Windows operating
     system. This attribute only exists on Windows machines.
 
+
 In addition, there are a few attributes that are automatically inserted
 into the machine ClassAd whenever a resource is in the Claimed state:
 
-:index:`ClientMachine<single: ClientMachine; ClassAd machine attribute (in Claimed State)>`
-
-``ClientMachine``
+:classad-attribute:`ClientMachine`
     The host name of the machine that has claimed this resource
 
-:index:`RemoteAutoregroup<single: RemoteAutoregroup; ClassAd machine attribute (in Claimed State)>`
-
-``RemoteAutoregroup``
+:classad-attribute:`RemoteAutoregroup`
     A boolean attribute which is ``True`` if this resource was claimed
     via negotiation when the configuration variable
-    ``GROUP_AUTOREGROUP`` :index:`GROUP_AUTOREGROUP` is ``True``.
+    ``GROUP_AUTOREGROUP`` :classad-attribute:`GROUP_AUTOREGROUP`.
```

Comment:
This is confused.  Needs move the GROUP_AUTOREGROUP index to before the entry and remove this :clasad-attribute and add back in "is ``True``".